Sources/CacheAPI-sqlite.php

```diff
@@ -24,8 +24,16 @@ class sqlite_cache extends cache_api
 	 * @var string The path to the current $cachedir directory.
 	 */
 	private $cachedir = null;
+
+    /**
+     * @var SQLite3
+     */
```

Comment:
This isn't the only place I see, but lots of spaces rather than tabs are being used.